protocols/1359-solvo-biotechnology/serial_dilution.ot2.py

```diff
@@ -0,0 +1,40 @@
+from opentrons import labware, instruments
+
+plate = labware.load('96-flat', '1')
+deep_plate = labware.load('96-deep-well', '2')
+tipracks = [labware.load('opentrons-tiprack-300ul', slot)
+            for slot in ['4', '5']]
+
+p50 = instruments.P50_Single(
+    mount='left',
+    tip_racks=tipracks)
+
+
+def run_custom_protocol(
+        special_mix_identifiers: str='A, B, 10',
+        special_mix_height: float=7,
+        dilution_transfer_volume: float=20,
+        deep_plate_transfer_volume: float=5):
+
+    # serial dilution
+    for row in plate.rows():
+        p50.transfer(
+            dilution_transfer_volume,
+            row.wells('12', to='2'),
```

Comment:
uhh, does this actually work?

I was under the impression that you can only index with specified well name. alternatively you can do:
```
row[11:1]
```

Reply:
I uploaded it to the app and it seems to work for me